cursor-target.js

```diff
@@ -33,6 +33,12 @@
  * addClickFunction(callback);
  * removeClickFunction(callback);
  *
+ * addDoubleClickFunction(callback);
+ * removeDoubleClickFunction(callback);
+ *
+ * addTripleClickFunction(callback);
+ * removeTripleClickFunction(callback);
```

Comment:
While I believe the double click is useful (albeit a "hidden" UX gesture, that is often not apparent through visuals and not popular in VR), the triple click is not a concept that users are familiar with and therefore would skip it. (In the long term adding too many things that aren't really needed would result in the JavaScript bundle getting unnecessarily large)

Could you please remove it for this pull request?